regression/contracts-dfcc/assigns-enforce-malloc-zero/test.desc

```diff
@@ -1,7 +1,7 @@
 CORE
 main.c
 --dfcc main --enforce-contract foo _ --malloc-may-fail --malloc-fail-null
-^\[foo.assigns.\d+\] line 19 Check that a\[\(signed long (long )?int\)i\] is assignable: SUCCESS$
+^\[foo_wrapped_for_contract_checking.assigns.\d+\] line 19 Check that a\[\(signed long (long )?int\)i\] is assignable: SUCCESS$
```

Comment:
For contracts, we should not refer to the wrapper function, but to the original function (i.e., `foo`). The user should identify the assignment in the original function that violates the assigns clause. We might want to keep the change from this PR, but fix this in our instrumentation. (cc. @remi-delmas-3000 )

Reply:
Actually, in our instrumentation we use the [from_expr_using_mode](https://github.com/diffblue/cbmc/blob/142db0bc2e2609538bfb68407ab770b93a71255e/src/langapi/language_util.cpp#L21-36) function.
```cpp
std::string from_expr_using_mode(
  const namespacet &ns,
  const irep_idt &mode,
  const exprt &expr)
{
  std::unique_ptr<languaget> language = (mode == ID_unknown)
                                          ? get_default_language()
                                          : get_language_from_mode(mode);
  INVARIANT(
    language, "could not retrieve language for mode '" + id2string(mode) + "'");

  std::string result;
  language->from_expr(expr, result, ns);

  return result;
}
```

Reply:
One way to fix this (but I might be missing something) is to rename the original function instead?!